### ISSUE TITLE
Basemap not imported correctly: matplotlib import error?

### ISSUE THREAD

**Issue by drphilmarshall** [fjaviersanchez/JupyterTutorial#17]
Not sure if this is a syntax or import error - it says the latter but I don't know the correct spelling. I'm getting this from the `binder` instance - do we need to add matplotlib to the `requirements.txt`, do you think?

``` python
from mpl_toolkits.basemap import Basemap
from matplotlib.collections import PolyCollection
```

> ---
> 
> ImportError                               Traceback (most recent call last)
> <ipython-input-58-9cdf36c23cfc> in <module>()
> ----> 1 from mpl_toolkits.basemap import Basemap
>       2 from matplotlib.collections import PolyCollection
> 
> ImportError: No module named basemap

**Comment by fjaviersanchez** [comment]
yes, we should add `mpl_toolkits` in `requirements.txt` (`conda install basemap`)

**Comment by drphilmarshall** [comment]
Go for it! :-)

**Comment by fjaviersanchez** [comment]
I added `RUN conda install basemap` on `Dockerfile`. Unfortunately the disk of the binder server was full when I tried to build the notebook and the process is halted. However, there is a built version in which everything works except for the `basemap` part. To make it work you can add before that cell `!conda install basemap -y` and it will install the package and run without any trouble. Could you please try if you can build it?

**Comment by drphilmarshall** [comment]
I think this can now be closed - see #18.
